### ISSUE TITLE
Crash on armhf when local.conf is empty

### ISSUE THREAD

**Issue by lberki** [zerotier/ZeroTierOne#979]
When I have an empty `local.conf`, ZeroTier crashes on my little Raspberry Pi 3B running Raspbian 10:

```
rm /var/lib/zerotier-one/local.conf
touch /var/lib/zerotier-one/local.conf
zerotier-one
```

Result is a segfault. If the file contains 

```
{
}
```

ZT comes up as expected. Various other things in the file also cause a crash. For example, this works:
```
{
  "physical": {
    "": {}
  }
}
```

But this one causes a crash:
```
{
  "physical": {
    "10.0.0.0/8": {}
  }
}
```

I'm running two networks, one on `my.zerotier.com` and one I host on my own server. `dpkg -l` says I'm running version `1.2.12`.

**Comment by adamierymenko** [comment]
Does this happen on the recently released 1.4.0?

**Comment by adamierymenko** [comment]
Never mind, able to reproduce
